src/components/amazon-polly/amazon-polly.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import get from "lodash/get";
+import { HeadsetIcon } from "./headset-icon";
+import { CloseIcon } from "./close-icon";
+
+export class AmazonPolly extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isOpen: false
+    };
+  }
+
+  onClickHandler = () => {
+    this.props.preProcess();
+    this.setState({
+      isOpen: true
+    })
+  }
+
+  closeHandler = () => {
+    this.setState({
+      isOpen: false
+    })
+  }
+
+  render() {  
+    const { caption, className = "amazon-polly", story, config, audioAttributes = { controls: true }, headsetColor = "#333", headsetSize = 24 } = this.props;
```

Comment:
`CloseIcon` props for `size` and `color` should also be passed down from here, right?